src/function/finitediff.jl

```diff
@@ -56,7 +56,9 @@ function instantiate_function(f, x, adtype::AutoFiniteDiff, p, num_cons=0)
     end
 
     if f.hess === nothing
-        hess = (res, θ, args...) -> FiniteDiff.finite_difference_hessian!(res, x -> _f(x, args...), θ, FiniteDiff.HessianCache(θ, adtype.fdhtype))
+        hesscache = FiniteDiff.HessianCache(x, adtype.fdhtype)
+        updatecache = (cache, x) -> (cache.xmm .= x; cache.xmp .= x; cache.xpm .= x; cache.xpp .= x; return cache)
```

Comment:
This should happen in the hessian call

Reply:
Yes, it's a function, it's getting called in the next line

Reply:
I mean, that should be in FiniteDiff.jl

Reply:
Oh lol, makes sense